packages/psss/src/containers/Tables/CountryReportTable.js

```diff
@@ -58,36 +87,57 @@ export const CountryReportTableComponent = React.memo(
 
     const handleSubmit = async () => {
       setTableStatus(TABLE_STATUSES.SAVING);
-      await onSubmit(fields);
+      await onSubmit({
+        ...fields,
+        sitesReported: parseInt(sitesReportedValue, 10),
+      });
       setTableStatus(TABLE_STATUSES.STATIC);
     };
 
     return (
       <LoadingContainer isLoading={tableStatus === TABLE_STATUSES.SAVING}>
-        <LayoutRow>
-          <Typography variant="h5">7/10 Sites Reported</Typography>
+        <FlexSpaceBetween pb={2}>
+          {tableStatus === TABLE_STATUSES.EDITABLE ? (
+            <FormRow>
+              <ReportedSites variant="h6">Reported Sites:</ReportedSites>
+              <StyledTextField
+                value={sitesReportedValue}
+                onChange={event => setSitesReportedValue(event.target.value)}
+                name="sites-reported"
+              />
+              <ReportedSites variant="h6"> / Total Sites: 10</ReportedSites>
+            </FormRow>
+          ) : (
+            <Typography variant="h5">7/{sitesReportedValue} Sites Reported</Typography>
```

Comment:
```suggestion
            <Typography variant="h5">{sitesReportedValue}/10 Sites Reported</Typography>
```
Though also the '10' should be coming from the back end at some point